packages/astro/components/shiki.ts

```diff
@@ -0,0 +1,46 @@
+import { type Highlighter, getHighlighter } from 'shikiji';
```

Comment:
Maybe we should move this file inside the `internal-helpers` package. Its content seems to be the same as we have in the markdoc integration.

Reply:
Yeah I'm planning to do a second round refactor once this is merged. It's in 3 places now. `shikiji` simplifies the usage a bit so I thought of duplicating the code like before for now, then refactor it as one again.